docs/README.rst

```diff
@@ -18,53 +18,79 @@ The |cfctlr-long| (|cfctlr|) manages F5 BIG-IP `Local Traffic Manager <https://f
 
-The |cfctlr-long| receives route updates and transforms them into BIG-IP policies when the following events occur in Cloud Foundry:
+The |cfctlr| receives route updates and transforms them into BIG-IP policy rules when the following events occur:
 
-- push, delete, and scale applications;
-- create, delete, map, and unmap routes.
+- push, delete, and scale applications (with mapped routes to the application);
+- map and unmap routes.
 
 For example:
 
-#. Developer pushes fooApp to Cloud Foundry.
-#. |cfctlr-long| discovers new route information for fooApp.
-#. |cfctlr-long| creates a pool and pool member(s) for each fooApp instance.
-#. |cfctlr-long| updates BIG-IP routing policy directing fooApp requests to the fooApp pool
-#. |cfctlr-long| monitors Cloud Foundry routing table and reconfigures the BIG-IP device when it discovers changes.
+#. Developer pushes "*myApp*" to Cloud Foundry (*without the CloudFoundry* ``--no-route`` *option*).
+#. |cfctlr| discovers new route information for *myApp*.
+#. Controller creates a pool and pool member(s) for each *myApp* instance.
+#. Controller updates BIG-IP routing policy directing App requests to the *myApp* pool.
+#. Controller monitors Cloud Foundry routing table and, when it discovers changes, reconfigures the BIG-IP device.
+
+The BIG-IP device handles traffic for every Cloud Foundry mapped route and load balances traffic to each application instance.
+The |cfctlr| can create a total of two (2) BIG-IP virtual servers for HTTP routes: one (1) for HTTP, one (1) for HTTPS.
+The Controller creates an additional virtual server for each TCP route.
 
-The BIG-IP device handles traffic for every Cloud Foundry route and application and load balances to each application instance.
-All route domains are global.
-The |cfctlr-long| can create a total of two (2) BIG-IP virtual servers:
+All HTTP traffic goes through either the port 80 or port 443 virtual server.
+The Controller routes TCP traffic through dedicated virtual servers configured to listen on non-HTTP ports.
 
-- one (1) for http
-- one (1) for https.
+You can attach the BIG-IP objects listed below to the HTTP virtual servers the |cfctlr| creates for CloudFoundry.
+To do so, create the desired objects on the BIG-IP manually *before* adding them to the Controller configuration in CloudFoundry.
 
-The virtual server contains pools and pool members for each application and application instance.
-You can define policies, profiles, and health monitors on the BIG-IP device in advance and apply them to the virtual server created by |cfctlr|.
+- policies
+- profiles
+- SSL profiles
+- health monitors
+
+The |cfctlr| also supports Cloud Foundry native health checks.
 
 .. _cfctlr-configuration:
 
 Configuration Parameters
 ------------------------
 
+The configuration parameters below customize the |cfctlr| behavior.
+Define the parameters in the ``env`` section your `application manifest`_ using the environment variable ``BIGIP_CTLR_CFG``.
```

Comment:
do we want to link to the solution doc here instead of the cloud foundry docs?

`/containers/v1/cloudfoundry/cfctlr-app-install.html`

Reply:
I think that's a good idea. I just want to be sure that there are really two configurations here. There are the cloud foundry required bits that the manifest documentation defines, and our environment parameter which delivers the configuration to the controller.